<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastaDialogBase</class>
 <widget class="QDialog" name="CadastaDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadasta QGIS plugin</string>
  </property>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>480</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="text">
    <string>Cadasta Project Download Wizard</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="step_1_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Step 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>491</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string>Your data is being downloaded. Once the download is complete, it will be added to yout project as a new memory layer.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="step_2_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Step 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Data download progress</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Your data is being downloaded</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress_bar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
